how_to/contribute_to_the_bpd_community.md

```diff
@@ -1 +1,92 @@
 # How to Contribute to the BPD Community
+Contributing to open source can be a rewarding experience, and getting started is easier than you might think. Here’s a beginner-friendly guide to help you navigate your way into the open source world:
+
+### 1. **Understand What Open Source Is**
+
+Open source software is developed collaboratively and made available to anyone to use, modify, and distribute. Contributions can range from fixing bugs and adding features to improving documentation.
+
+### 2. **Find a Project**
+
+#### **Explore Repositories**
+
+- **GitHub**: Popular platform where many open source projects are hosted. Explore repositories through the [Explore page](https://github.com/explore) or by searching for topics that interest you.
+- **GitLab**: Another platform similar to GitHub. Look for projects via [GitLab Explore](https://gitlab.com/explore/projects).
+- **Bitbucket**: Also hosts open source projects. Check out repositories via [Bitbucket’s Explore page](https://bitbucket.org/product).
+
+#### **Look for Good First Issues**
+
+Many projects label issues that are suitable for beginners. Search for labels like “good first issue,” “beginner,” or “help wanted.”
+
+### 3. **Understand the Project**
+
+#### **Read the Documentation**
+
+- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the read me first: https://github.com/CoderBlee/blackpythondevs
+- **CONTRIBUTING.md**: Guidelines for contributing to the project. This can include coding standards, testing protocols, and how to submit changes.
+- **CODE_OF_CONDUCT.md**: Code of conduct outlines the expectations for behavior in the community.
+
+#### **Set Up Your Development Environment**
+
+Follow the setup instructions provided in the documentation. This may involve installing dependencies, setting up a development environment, or running tests.
+
+### 4. **Start Contributing**
+
+#### **Fork and Clone the Repository**
```

Comment:
This Section should likely come before the setup your environment

Reply:
I was not well and have exams, I saw that someone already have made the changes. Should I still do the changes you have requested or look into other issues

Reply:
I haven't seen if all the changes have been made. I would keep your code and address the issues with individual PRs for anything still active.

Reply:
Can I still just implement what you said and make the changes regardless?